packages/p2p/src/components/my-ads/my-ads-table.jsx

```diff
@@ -26,7 +28,37 @@ const getHeaders = offered_currency => [
 const MyAdsTable = () => {
     const { floating_rate_store, general_store, my_ads_store } = useStores();
     const [selected_advert, setSelectedAdvert] = React.useState(undefined);
-    const local_currency = general_store.client.local_currency_config.currency;
+
+    const generateHintBoxMessage = () => {
```

Comment:
Could the generateHintBoxMessage be placed outside of the parent component `MyAdsTable`? Or alternatively we could create a new component called like `HintBoxMessage` within the `my-ads-table.jsx`, this is to avoid the parent component mounting and re-mounting the declared component returned in this function.
https://levelup.gitconnected.com/code-review-avoid-declaring-react-component-inside-parent-component-1768a645f523

Reply:
Refactored code